CHANGELOG.md

```diff
@@ -16,6 +16,10 @@
     - If a legacy embedding code is detected it will show the link to the presentation instead of the embedded presentation
 - Speakerdeck embedding
     - If a legacy embedding code is detected it will show the link to the presentation instead of the embedded presentation
+- Code highlighting for the follow languages
+    - tiddlywiki
+    - mediawiki
+    - jsx
```

Comment:
Do we give some explanation? I mean I can live without, but maybe that would be a good idea?

Reply:
Good point. 

Extended